rust/kernel/str.rs

```diff
@@ -11,7 +11,69 @@ use crate::c_types;
 /// Byte string without UTF-8 validity guarantee.
 ///
 /// `BStr` is simply an alias to `[u8]`, but has a more evident semantical meaning.
```

Comment:
This change makes it not an alias to `[u8]` but a newtype wrapper. This part of the documentation should be changed.

Reply:
Thank you, done!